libs/blocks/tabs/tabs.js

```diff
@@ -99,7 +99,7 @@ const handleDeferredImages = (block) => {
 }
 
 const init = (block) => {
-  const rootElem = block.closest('.fragment') || document;
+  const rootElem = block.closest('.georouting-wrapper.fragment') || document;
```

Comment:
I remember that this change was done because tabs weren't loading properly within some fragments, so I'm a little worried to just change this without doing some thorough testing. 
Addtionally, I do not think it's a good change to have georouting specific selectors within the tabs block. Blocks should be independent of one another, so doing this looks like a bandage of the problem and is leading to an anti-pattern. 
I'd like to have more time to look, but don't really have any  atm. 
Why don't we just fix the selector on line 183 which is causing the current problem ? 
`assocTabItem = rootElem.querySelector(`#tab-panel-${id}-${val}`);`

Reply:
Thanks for the review @vhargrave . I agree to your point blocks should be independent. Only reason I did this change because there was already a block (fragment) added here so with existing approach we already have that block dependency issue. Also I could only find georouting modal working with existing fix hence the change all the other fragments I checked were broken with tab. Please do share if you remember which other fragments needed the initial fix. In the meantime I have gone ahead and updated the selector. Please review and let me know if any concern.  We might also need to fix the other issue of dependency of 2 blocks (tab and fragment) which already exists (not related to this PR).

Reply:
If I get some time to look I'll send them on. 
the change now looks good to me though, thanks!